docs/sphinx/user_guide/feature/index.rst

```diff
@@ -0,0 +1,63 @@
+.. ##
+.. ## Copyright (c) 2016-17, Lawrence Livermore National Security, LLC.
+.. ##
+.. ## Produced at the Lawrence Livermore National Laboratory
+.. ##
+.. ## LLNL-CODE-689114
+.. ##
+.. ## All rights reserved.
+.. ##
+.. ## This file is part of RAJA.
+.. ##
+.. ## For details about use and distribution, please read RAJA/LICENSE.
+.. ##
+
+.. _index-label:
+
+====================
+Indices and Segments
+====================
+
+Loop counters and iteration spaces are fundamental to C/C++ loops. In this section we introduce RAJA specific index types 
+and iteration space which are designed to encapsulate common programming patterns.
+
+.. note:: * All segment objects and the ``Index_type`` are found in the namespace RAJA
+
+-------
+Indices
+-------
+
+Although RAJA may utilize any loop counter, the recommended loop counter is the ``RAJA::Index_type``. The ``RAJA::Index_type`` 
```

Comment:
Saying `loop counter type` would be clearer than just loop counter (which could refer to the instance of the variable).